components/card-list/css/_mixin.scss

```diff
@@ -1,6 +1,6 @@
 @mixin utrecht-card-list {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(var(--utrecht-card-list-item-max-inline-size, 350px), 1fr));
```

Comment:
Ik denk dat deze 350px niet hardcoded zou moeten zijn. Het zou een CSS variable voor een design token moeten zijn. Als er een fallback nodig is, dan zou ik zeggen: `1fr` of iets zinnigs in `em`.